src/FiltrationConditionComponent/FiltrationConditionComponent.tsx

```diff
@@ -22,8 +22,8 @@ export class FiltrationConditionComponent extends React.Component<Props> {
 		return (
 			<li className="kit-filtration-condition">
 				<span className="kit-filtration-condition__item-text">
-					<b>{filtrationObjectName}</b>
-					{!!filtrationMethodName && <>{filtrationMethodName} </>}
+					<b>{`${filtrationObjectName} `}</b>
+					{!!filtrationMethodName && `${filtrationMethodName} `}
```

Comment:
Восклицательные знаки выглядят конечно красиво, но лучше использовать более явное и простое `filtrationMethodName !== undefined`, так быстрее разобраться

Reply:
Либо вообще, если нам не нужно будет выводить пустую строку, можно просто:
```suggestion
					{filtrationMethodName && `${filtrationMethodName} `}
```